server/odc-service/src/main/java/com/oceanbase/odc/service/notification/helper/EventBuilder.java

```diff
@@ -188,6 +189,7 @@ private Event ofTask(TaskEntity task, TaskEvent status) {
         labels.putIfNonNull(CREATOR_ID, task.getCreatorId());
         labels.putIfNonNull(TRIGGER_TIME, LocalDateTime.now().format(DATE_FORMATTER));
         labels.putIfNonNull(REGION, OB_ARN_PARTITION);
+        labels.putIfNonNull(DESCRIPTION, task.getDescription());
```

Comment:
Is this change related to the issue?

Reply:
No, this is additional code, just added to this PR by the way~

Reply:
Why do we need this label?